fastlane/jetpack_metadata/ko/subtitle.txt

```diff
@@ -1 +1 @@
-WordPress 사이트를 더 빠르고 더 안전하게 만드세요.
\ No newline at end of file
+더 빠르고 더 안전한 워드프레스
```

Comment:
17 characters ✅

That is if App Store Connect does something like "...".length, which would make sense to me given that the 30 characters limitation is not a matter of sentence complexity but of screen real estate. I'm not actually sure if Korean characters are counted in the same way as we do with the letters in a Roman language.